lib/ruby_lsp/requests/support/dependency_detector.rb

```diff
@@ -64,7 +64,12 @@ def detect_typechecker
       return false if ENV["RUBY_LSP_BYPASS_TYPECHECKER"]
 
       Bundler.with_original_env do
-        Bundler.locked_gems.specs.any? { |spec| spec.name == "sorbet-static" }
+        sorbet_static_detected = Bundler.locked_gems.specs.any? { |spec| spec.name == "sorbet-static" }
+        # Don't show message while running tests, since it's noisy
+        if sorbet_static_detected && !defined?(Minitest)
```

Comment:
I wondering if there's a better way... I could do `ENV["CI"]` but it would still show locally.

Reply:
Yeah it's not ideal... any thoughts @st0012?